src/ToDo.UI/Views/TaskListPage.xaml

```diff
@@ -88,11 +88,13 @@
 												<utu:AutoLayout PrimaryAxisAlignment="Center" utu:AutoLayout.CounterAlignment="Start">
 													<TextBlock Foreground="{ThemeResource OnSurfaceBrush}"
 															   Text="{Binding Title}"
+															   MaxLines="1"
```

Comment:
This is not part of the generated code at the moment if I'm not mistaken, right?
I know it's a quick fix that you want to implement to look better, but in this type of case should we not have a discussion with @NVLudwig and @AntonioBonillaHorus? 
(I know maybe it's not an urgent priority compared to other things at the moment but at least raising it so the proper steps could be done later on).
Do we need to have an example in the Figma style with bigger values so the plugin can generate these properties (maxlines, trimming, ect...) ? Are there any known issues regarding the plugin in order to generate the proper code and if so do we need to open an issue on the Figma plugin repo ?

Current Microsoft To-Do app for example it's more than 1 line:
![image](https://user-images.githubusercontent.com/16295702/168098861-9e7eae8d-b639-4d1c-99c2-146cf1e6a939.png)

Reply:
@NVLudwig, @Xiaoy312, @carldebilly for now maybe we can have a discussion on the issue here https://github.com/unoplatform/uno.todo/issues/140 regarding what we decide for now for this.
(Deciding what will be the behavior for this in the app (matching Microsoft To-Do ?), fixing it in the app and opening a figma plugin issue to fix it later on for the generated code?)